back/controllers/palController.js

```diff
@@ -32,20 +32,28 @@ exports.getOnePal = async (req, res) => {
     });
     res.json(pal);
   } catch (error) {
-    console.error('Error getting pal:', error);
-    res.status(500).json({ error: 'Internal server error' });
+    console.error("Error getting pal:", error);
+    res.status(500).json({ error: "Internal server error" });
   }
 };
 
 exports.createPal = async (req, res) => {
-  const { number, name, url, elements, partnerSkill, worksFor, drops } = req.body;
+  const { number, name, url, elements, partnerSkill, worksFor, drops } =
+    req.body;
   try {
     const pal = await prisma.pal.create({
       data: {
-        number, name, url, elements: { connect: elements.map((element) => ({ idElement: element.id })) },
+        number,
+        name,
+        url,
+        elements: {
+          connect: elements.map((element) => ({ idElement: element.id })),
+        },
         partnerSkill: { connect: { idPartnerSkill: partnerSkill } },
-        worksFor: { connect: worksFor.map((WorkFor) => ({ idWorksFor: WorkFor.id })) },
-        drops: { connect: drops.map((Drop) => ({ idDrop: Drop.id })) }
+        worksFor: {
+          connect: worksFor.map((WorkFor) => ({ idWorksFor: WorkFor.id })),
```

Comment:
Try to avoid using Pascal Casing and instead prefer Camel Casing in this `map()` method.

Reply:
C'est impeccable